test/Common/DurableClientBaseTests.cs

```diff
@@ -166,7 +166,7 @@ public async void DurableClient_ExternalApp_GetStatusAsync_ReturnsStatus()
             orchestrationServiceClientMock.Setup(x => x.GetOrchestrationStateAsync(It.IsAny<string>(), It.IsAny<bool>()))
                 .ReturnsAsync(GetInstanceState(OrchestrationStatus.Running));
 
-            var durableOrchestrationClient = this.GetDurableClient(orchestrationServiceClientMock.Object); 
```

Comment:
Surprised we had some whitespace in the codebase. Don't we have linter rules against it?

Reply:
We do, but they are warnings, and warnings currently only fail in dev, because some of our tests purposely have bad practices.

It may be worth explicitly ignoring those warning, and making warnings errors in test code as well.

Reply:
Filed issue for it here: https://github.com/Azure/azure-functions-durable-extension/issues/1583